omf-bifrost/Cargo.toml

```diff
@@ -40,6 +40,9 @@ assert_cmd = "2.0"
 predicates = "3.0"
 tempfile = "3.8"
 
+# Examples
+duckdb = { version = "1.2.2", features = ["bundled"] }
```

Comment:
I'm using the "bundled" flag here, which causes a full build of the duckdb library. This is probably fine for now to make it easy, but ideally we turn this off to make it faster to compile (linked to system library).